src/dotty/tools/dotc/core/Symbols.scala

```diff
@@ -417,14 +417,6 @@ object Symbols {
     /** If this symbol satisfies predicate `p` this symbol, otherwise `NoSymbol` */
     def filter(p: Symbol => Boolean): Symbol = if (p(this)) this else NoSymbol
 
-    /** Is this symbol a user-defined value class? */
-    final def isDerivedValueClass(implicit ctx: Context): Boolean = {
```

Comment:
Yes, I wasn't sure about that, it seems nice to keep all the value class methods in `ValueClasses.scala`, but I can move that method in `Symbols` if you think that's better.